src/components/CldImage/CldImage.vue

```diff
@@ -7,7 +7,7 @@
 <script>
 import { Cloudinary, Transformation } from "cloudinary-core";
 import { merge, range } from "../../utils";
-import {accessibilityTransformations} from '../../constants';
+import {A11Y_TRANSFORMS, COMPONENTS, PLACEHOLDER_OPTIONS } from '../../constants';
```

Comment:
Why are placeholders considered "options" and a11y are transformations?

could we rename placeholder to PLACEHOLDER_TRANSFORMATIONS?